sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -0,0 +1,281 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
```

Comment:
Just noticing this - why? I think any suppressions should be localized

Reply:
@yeandy

Reply:
Looks like there was an [issue (BEAM-9064)](https://issues.apache.org/jira/browse/BEAM-9064) to add `pytype` checks to our lint checks. One of the prerequisite [tasks (BEAM-9071)](https://issues.apache.org/jira/browse/BEAM-9071) was to ignore pytype until the typing feature was fully developed. [(BEAM-9071)](https://issues.apache.org/jira/browse/BEAM-9071) was completed, which populated all of our python files with `# pytype: skip-file`, but it seems that [issue (BEAM-9064)](https://issues.apache.org/jira/browse/BEAM-9064) was never done as the PR was closed.

And looks like `# pytype: skip-file` is present in both main module files as well as test files.

@tvalentyn How should we handle this `# pytype: skip-file`? Maybe keep it for now until we have bandwidth to clean up our linting/typing processes?

Reply:
Sounds not related,  I'll start a discussion off this PR. Thanks